recipesearch/src/Recipes/Root/Root.js

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react'
 import { SearchBar }from '../SearchBar';
+import { SearchResults }from '../SearchResults';
```

Comment:
prettify this by adding a space after the first `}`

Reply:
Let me work on this.

Reply:
I have used prettify to format the document